Makefile

```diff
@@ -82,11 +82,13 @@ lint-python:
 	@python3 -m ruff format --check ./
 # Lint HTML files, and the build output of the docs
 lint-html:
-	@yarn htmlhint "testCourse/**/question.html" "exampleCourse/**/question.html" "site"
+	@yarn htmlhint "testCourse/**/question.html" "exampleCourse/**/question.html" "site" || \
```

Comment:
If the package is installed, use it. Otherwise, download the same exact version on-the-fly using the local metadata.

Reply:
When would these packages not be installed? They're listed in `package.json`, right?

Reply:
https://github.com/PrairieLearn/PrairieLearn/pull/11909#discussion_r2073997188